assets/js/vue/components/search/SearchForm.vue

```diff
@@ -76,7 +76,7 @@
                 @noOfResults="changeNoOfResults"
                 :formValues="form"/>
         <section class="section-content pt-3 bg" v-else>
-            <div class="container">
+            <div v-show="!hideSearchTextBlock" class="container">
```

Comment:
using the same name as the window variable key is kind of confusing. Should we name the function something else?

Reply:
```suggestion
            <div v-show="!displayTextBlock" class="container">
```